aerofiles/seeyou/reader.py

```diff
@@ -193,3 +193,90 @@ def decode_description(self, description):
             return None
 
         return description
+
+    def decode_taskname(self, fields):
+        if not fields:
+            raise ParserError('Name field must not be empty')
+
+        return fields[0]
+
+    def decode_tasklist(self, fields, waypoints):
```

Comment:
> the implementation of a reader method was really straight forward.

that's what I was hoping for 😉 

> reason i want to edit the converter first is that i can check with tests before i commit

why can't you test without changing the converter?